packages/core-modules/contracts/interfaces/IElectionModule.sol

```diff
@@ -20,15 +20,23 @@ interface IElectionModule {
 
     function setSeatCount(uint seats) external;
 
-    function setEpochDuration(uint duration) external;
+    function setEpochDuration(uint64 duration) external;
 
     function setPeriodPercent(uint8 percent) external;
 
     function setNextSeatCount(uint seats) external;
 
-    function setNextEpochDuration(uint duration) external;
+    function setNextEpochDuration(uint64 duration) external;
 
     function setNextPeriodPercent(uint8 percent) external;
 
     function elect(address[] memory candidates) external;
+
+    function epochFinised() external view returns (bool);
+
+    function isEpochNominationPeriod() external view returns (bool);
+
+    function isEpochVotingPeriod() external view returns (bool);
+
+    function initialize() external;
```

Comment:
I don't know if will be needed. Potentially yes, but at least needed for testing and we need to cover the Epoch 0 case.

Reply:
Ok, but if it is needed, this name won't do because of the router proxy's requirement of unique selectors. It would probably need to be named `initializeElectionModule` or something.

Reply:
Changed for something that makes more sense in the naming, since is not initializing, but setting up the Epoch Zero (initial epoch)

Reply:
Are we protecting overwrites in these new setup functions?